magda-web-client/src/assets/spinner.svg

```diff
@@ -0,0 +1,49 @@
+<svg class="lds-spinner" width="32px"  height="32px"  xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" style="background: none;"><g transform="rotate(0 50 50)">
```

Comment:
@AlexGilleran this one means to be an inline place holder~ See below:
![image](https://user-images.githubusercontent.com/674387/58524163-afaac900-820a-11e9-9def-64db846c1b3c.png)
It could be multiple independent indicators on screen at the same time~
Existing **square** one (see below), in my opinion, doesn't look right 😄 and probably too `heavy`(It's not an image). 
![image](https://user-images.githubusercontent.com/674387/58524268-031d1700-820b-11e9-9e5c-7e5f5479ca3b.png)
If you prefer the square one, just let me know~

Reply:
Ah cool beans :)